spa/src/pages/ConfigSteps/index.tsx

```diff
@@ -183,6 +185,7 @@ const ConfigSteps = () => {
 			case 1: {
 				setLoaderForLogin(true);
 				try {
+					analyticsClient.sendUIEvent({ actionSubject: "authorizeButton", action: "clicked" });
```

Comment:
I think the name of the action should be more specific here.
```suggestion
					analyticsClient.sendUIEvent({ actionSubject: "authorizeToGitHubCloud", action: "clicked" });
```